templates/instruc_template.forTasks.rst

```diff
@@ -0,0 +1,236 @@
+:orphan: true
+	 
+.. Based on: https://dmtn-030.lsst.io/v/DM-7096/index.html#task-topic-type, with learnings from the 4 sfp pages built in branch DM-8717
+
+########################
+Guidance for ExampleTask
+########################
+
+We give guidance here for developers on what to write into the
+reStructured Text (reST) `Task Template
+<struc_template.forTasks.html>`_ which, when processed by the
+documentation-building code, will make the primary page for that Task.
+
+Some sections are fully automatically populated from python docstrings
+in the code and will not require developer input, and this information
+is also copied into the API page for this Task.  We demarcate those
+sections as well and guidance for how to write those docstrings is
+covered in `this section of the Developer Guide
+<https://developer.lsst.io/docs/py_docs.html>`_ .
+
+Below, the normal text lists notes on the each of the sections of the
+template, some of which is repeated from the `Task Template
+<struc_template.forTasks.html>`_, but here with more specific
+direction to the developer of what to include into the reST for that
+section.
+
+To be explicit, we will give some examples for what to put into the
+sections taken primarily from `ProcessCcd` specifically, but from other tasks as well.
+
+[NB: The structure is substantially different from how docstrings were
```

Comment:
Ok - i find n.b. pretty standard and shorthand usage, but in any case we don't really need to say anything like this at all, here.  I'd wondered if it might be seen to be useful, thought it could be because many of the devs will be familiar with how the doxygen pages were written and we can let them know now that the new 'regime' will be different but fine to keep it leaner and leave this out.